setup.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import functools
+from setuptools import setup
+
+
+def read(filename):
+    try:
+        with open(filename, 'r', encoding='utf-8') as f:
+            return f.read()
+    except IOError:
+        return ''
+
+
+readme = functools.partial(read, 'README.rst')
+
+
+setup(
+    name='prometheus-gatling-exporter',
+    author='Andrei Dan',
+    author_email='andrei@crate.io',
+    url='https://github.com/andreidan/prometheus-gatling-exporter',
+    description='A tool that parses Gatling live logs and exports metrics for\
+            Prometheus to scrape.',
+    long_description=readme(),
+    entry_points={
+        'console_scripts': []
```

Comment:
Could you add a console script?

`gatling_exporter = gatling.gatling_exporter:main`